CVLByExample/ExtensionContracts/ExtensionContracts.spec

```diff
@@ -14,11 +14,12 @@ rule setInExtension1Direct(env e) {
     assert base1.inExtension1 == n;
 }
 
+// the summarization of Extension1.getInExtension1() is applied
```

Comment:
MUST: This comment should stay with the `assert` - I think this is a comment case as well and we should allow comments to be attached to `asserts`